protocol-definitions/MC.yaml

```diff
@@ -955,3 +955,12 @@ methods:
           doc: |
             true if the session is found & closed,
             false otherwise.
+  - id: 37
+    name: demoteDataMember
```

Comment:
can we have a more descriptive name if it is demoting to a lite member, e.g.  `changeToLiteMember` or `demoteToLiteMember` ?

Reply:
it's consistent with the existing `promoteLiteMember`